db/seeds.rb

```diff
@@ -82,13 +82,13 @@
 end
 
 # Create subjects
-# TODO: this method will be created with actual subject data after the Publish Api data is integrated
-Subject.upsert_all([{ subject_area: "primary", name: "Primary with English", code: "01" },
-                    { subject_area: "primary", name: "Primary with geography and history", code: "02" },
-                    { subject_area: "secondary", name: "Biology", code: "C1" },
-                    { subject_area: "secondary", name: "Classics", code: "Q8" }])
+PublishTeacherTraining::Subject::Import.call(perform_action: :perform_now)
 
 # Create placements
-placement = Placement.create!(school: Placements::School.first, start_date: 1.month.from_now, end_date: 2.months.from_now)
-PlacementSubjectJoin.create!(placement:, subject: Subject.first)
-PlacementMentorJoin.create!(placement:, mentor: Placements::Mentor.first)
+Placements::School.find_each do |school|
+  placement = Placement.create!(school:, start_date: 1.month.from_now, end_date: 2.months.from_now)
+
+  subject = school.phase == "Primary" ? Subject.primary.first : Subject.secondary.first
+  PlacementSubjectJoin.create!(placement:, subject:)
+  PlacementMentorJoin.create!(placement:, mentor: Placements::Mentor.first)
```

Comment:
This hasn't changed since the previous implementation, but because of the start_date and end_date being `x.months.from_now` it will always create new placement when seeds are run. 
An alternative would be to check if any placements pre-exist for each school.
i.e `next if school.placements.any?`

Reply:
OK, if this hasn't changed it's fine